README.md

```diff
@@ -90,34 +95,27 @@ If you want to use any of the bundled icons, you need to add the icon fonts to y
     <string>Fontisto.ttf</string>
   </array>
   ```
-  
-</details>
 
-<br>
+After that, go to **XCode Settings -> Build Phases -> Copy Bundle Resources** and include the fonts you added.
```

Comment:
Automatically added to _[CP] copy pods resources_  ( The line not necessary ) ( react-native: 0.63.4 | react-native-vector-icons 7.1.0 )

Reply:
Great! I'll remove that.

Reply:
it doesnt help 

i tired from your stupid library

go find something else